eetfestijn/settings.py

```diff
@@ -80,7 +80,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'NAME': os.path.join(BASE_DIR, 'db', 'db.sqlite3'),
```

Comment:
I'm assuming this is to make writing a dockerfile easier, but could you clarify that in the PR description?

Reply:
ah too late